### ISSUE TITLE
Appveyor should send a success status as soon as all required jobs finish

### ISSUE THREAD

**Issue by deivid-rodriguez** [appveyor/ci#1960]
I have 6 required jobs and 1 allowed failure [in my appveyor matrix](https://github.com/deivid-rodriguez/byebug/blob/80b2c23d5065a4276957c8f891905e6543b29c24/appveyor.yml).

The allowed failure is the slowest entry (takes almost 1 hour vs the 1-2 minutes of the other entries). It happens that even if all the other jobs are green, a success status is not sent to Github until that slow entry finishes, even though it's not necessary for the status calculation.

I'd love that the CI status would be sent as soons as all required jobs are finished.

**Comment by IlyaFinkelshteyn** [comment]
Did you consider to create separate AppVeyor project against the same repo, which will build only `ruby_version: _trunk` as a workaround? For this project you can set `Custom commit status context` (UI-only settings on the General tab) so you can distinguish statuses set by this build.

**Comment by deivid-rodriguez** [comment]
Thanks @IlyaFinkelshteyn, it could work as a workaround indeed, although I would preferred that appveyor fixed this.

I would have to create a separate configuration file as well, right?

**Comment by IlyaFinkelshteyn** [comment]
Yes, you need separate configuration file. You can keep them side by side with [alternative YAML file name or location](https://www.appveyor.com/docs/build-configuration/#yaml-file-alternative-naming).
